revenue_app/utils.py

```diff
@@ -1,20 +1,40 @@
+import numpy as np
 import pandas as pd
 
 
-# transactions_w_pp = pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=1, header=1, usecols=range(0,16))
-# transactions_w_email = pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=2, header=1, usecols=range(0,16))
-
 def get_rollups():
-    return pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=0, header=1, usecols=range(0,53))
+    return pd.read_excel('datasets/Revenue Queries.xlsx', sheet_name=0, header=1, usecols=range(0, 53))
 
 
 def get_transactions():
-    return pd.read_csv('datasets/transactions.csv')
+    transactions = pd.read_csv('datasets/transactions.csv').replace(np.nan, '', regex=True)
```

Comment:
I don't see the point of having the path of these file hardcoded in here. Is there a particular reason you're doing it this way?

Reply:
We can hardcode it in another place, but until we get access to Presto we need to have it hardcoded somewhere. Ideally we should change it to get the query result